framework/src/org/checkerframework/framework/util/typeinference/DefaultTypeArgumentInference.java

```diff
@@ -370,7 +425,8 @@ private void clampToLowerBound(
             final AnnotatedTypeFactory typeFactory,
             final List<AnnotatedTypeMirror> argTypes,
             final AnnotatedExecutableType methodType,
-            final Set<TypeVariable> targets) {
+            final Set<TypeVariable> targets,
+            boolean useNullArguments) {
```

Comment:
Some documentation would be good for the new parameter. checker-framework-inference is using this method and fails to compile with this change.